operator/src/main/java/oracle/kubernetes/operator/helpers/PodHelper.java

```diff
@@ -85,6 +86,59 @@ public static boolean isReady(V1Pod pod) {
     return ready;
   }
 
+  /**
+   * Get list of scheduled pods.
+   * @param info Domain presence info
+   * @return list containing scheduled pods
+   */
+  public static List<String> getScheduledPods(DomainPresenceInfo info, String clusterName) {
```

Comment:
I'll verify below, but it's surprising that you have a method to list scheduled pods. I would think that you would be interested in unscheduled pods.